specification/network/resource-manager/Microsoft.Network/stable/2019-09-01/networkWatcher.json

```diff
@@ -607,6 +607,249 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/networkWatchers/{networkWatcherName}/flowLogs/{flowLogName}": {
+      "put": {
+        "tags": [
+          "FlowLogs"
+        ],
+        "operationId": "FlowLogs_CreateOrUpdate",
+        "description": "Create or update a flow log for the specified network security group.",
+        "parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group."
+          },
+          {
+            "name": "networkWatcherName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the network watcher."
+          },
+          {
+            "name": "flowLogName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the flow log."
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/FlowLogRequest"
```

Comment:
You should use the same definition for `PUT` and `GET`, so please use `FlowLog` and remove `FlowLogRequest`